src/app/libp2p_helper/src/libp2p_helper/main.go

```diff
@@ -847,6 +848,7 @@ func (ap *beginAdvertisingMsg) run(app *app) (interface{}, error) {
 	// report local discovery peers
 	go func() {
 		for info := range l.FoundPeer {
+			app.P2p.Host.Peerstore().AddAddrs(info.ID, info.Addrs, peerstore.ConnectedAddrTTL)
```

Comment:
oof, this regression happened because the `foundPeer` handler no longer tries to make a new connection.